.docker/scripts/create-test-data.sh

```diff
@@ -97,7 +97,7 @@ add_user_if_needed walker "Walker" walker@localhost
 
 # Create test dataset with our standard fields
 curl -LsH "Authorization: ${API_KEY}" \
-    --data '{"name": "test-dataset", "owner_org": "'"${TEST_ORG_ID}"'",
+    --data '{"name": "test-dataset", "owner_org": "'"${TEST_ORG_ID}"'", "private": true,
```

Comment:
do we need another one for public public?

Reply:
We can do that if we decide a test needs it. The sample data like Anna Karenina generally handles public access though.